frontend/client-list/toolbar/clients-table-toolbar.component.tsx

```diff
@@ -90,6 +167,37 @@ const css = `
   align-items: center;
 }
 
+& .bulk-action-icon {
+  height: 2rem;
+  margin: 0 1rem 0 -0.5rem;
+}
+
+& .bulk-action-menu-modal {
```

Comment:
Did the styleguide's popup class not work for you? Why did you recreate it? https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/master/frontend/styleguide.component.tsx#L190